tests/test_modeling_tf_common.py

```diff
@@ -152,7 +175,12 @@ def test_saved_model_with_hidden_states_output(self):
                 tf.saved_model.save(model, tmpdirname)
                 model = tf.keras.models.load_model(tmpdirname)
                 outputs = model(inputs_dict)
-                output = outputs[list(outputs.keys())[-1]] if isinstance(outputs, dict) else outputs[-1]
+
+                if self.is_encoder_decoder:
```

Comment:
@sgugger @LysandreJik @jplu - this slow test was failing for all models - fixed it here.